data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	As the crescendo of her impassioned statement crests to its conclusion there is a growing rumble of ascent and the room is suddenly filled with the angry cries of those both emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen elders jump to their feet but the cacophony is not so easily restrained. At your side Sayari's expression is ashen but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
+			`	There is a shift in the sound of the room and your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound though. A ripple of silence flows outward from his position and elders all around promptly drop back into their seats as it reaches them, leaving a scant handful who were too slow to react to have their final words trail off awkwardly. Only the last speaker remains on her feet, the moderately weakly announcing what you know to be, "The council hears Aimet."`
+			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so Gemnali but, unlike you, I was there when it was added to the mandate of this chamber."`
+			`	He peers around the room and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`
+			`	He coughs again it echoes in the silence of nearly a hundred Hai holding their breath. When he continues his voice has grown stronger and more cutting. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`
```

Comment:
```suggestion
			`	He coughs again, echoing in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman, who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`
```

Reply:
I think `He coughs again, and it echoes` would be better still, but probably a good catch on the switch to perfect present tense halfway there.
Not a fan of the comma after woman though.